resources/dicts/events/death/general.json

```diff
@@ -1722,54 +1790,43 @@
                     "m_c"
                 ],
                 "values": [
-                    "platonic"
+                    "dislike"
+                ],
+                "amount": 15
+            },
+            {
+                "cats_from": [
+                    "r_c"
+                ],
+                "cats_to": [
+                    "m_c"
+                ],
+                "values": [
+                    "platonic",
+                    "trust",
+                    "comfort"
                 ],
-                "amount": -5
+                "amount": -15
             }
         ]
     },
     {
-        "event_id": "gen_death_",
-        "camp": [
-            "any"
-        ],
-        "season": [
-            "any"
-        ],
-        "tags": [
-            "war"
-        ],
-        "weight": 20,
-        "event_text": "m_c was killed by a falling branch during a battle in a heavily wooded area of the forest.",
-        "m_c": {
-            "age": [],
-            "status": [],
-            "dies": true
-        },
-        "history": {
-            "reg_death": "m_c was squished by a falling branch.",
-            "lead_death": "was squished by a falling branch."
-        }
-    },
-    {
-        "event_id": "gen_death_",
+        "event_id": "gen_death_murder_multiice1",
+        "biome": ["any"],
         "camp": [
             "any"
         ],
         "season": [
             "newleaf",
             "leaf-bare"
         ],
-        "tags": [
-            "murder",
-            "revealed"
-        ],
+        "sub_type": [ "murder" ],
+        "tags": [],
         "weight": 20,
         "event_text": "m_c agreed to venture with r_c out onto a frozen pond. m_c went first while r_c hung back. m_c slid around playfully, taunting the other cat to chase {PRONOUN/m_c/object}. {PRONOUN/m_c/subject/CAP} {VERB/m_c/were/was} about to launch another teasing remark when the ice cracked below {PRONOUN/m_c/poss} paws. r_c lingered to make sure {PRONOUN/m_c/subject} didn't surface before running back to camp with a yowl.",
         "m_c": {
-            "age": [],
-            "status": [],
-            "relationship_status": [],
+            "age": [ "adolescent", "young adult", "adult", "senior adult" ],
+            "status":  ["apprentice", "mediator apprentice", "medicine cat apprentice", "warrior", "medicine cat", "mediator", "deputy"],
```

Comment:
we could maybe include this as an "all_lives" leader death too?  after all, if you're stuck under the surface of the ice it really won't do you much good to be revived.